src/frequenz/client/base/client.py

```diff
@@ -32,12 +29,31 @@ class BaseApiClient(abc.ABC, Generic[StubT]):
         a class that helps sending messages from a gRPC stream to
         a [Broadcast][frequenz.channels.Broadcast] channel.
 
+    Note:
+        Because grpcio doesn't provide proper type hints, a hack is needed to have
+        propepr async type hints for the stubs generated by protoc.  When using
+        `mypy-protobuf`, a `XxxAsyncStub` class is generated for each `XxxStub` class
+        but in the `.pyi` file, so the type can be used to specify type hints, but
+        **not** in any other context, as the class doesn't really exist for the Python
+        interpreter. This include generics, and because of this, this class can't be
+        even parametrized using the async class, so the instantiation of the stub can't
```

Comment:
```suggestion
        even parameterized using the async class, so the instantiation of the stub can't
```

Reply:
I found it confusing :D
I seems I was wrong :angel:

Reply:
it is an irregular language.

Reply:
I just noticed that it says `parametrized` when defining `past tense` and `past participle`, but uses `parameterized` in the example :rofl: